package.json

```diff
@@ -58,7 +58,7 @@
     "setup-https": "mkcert -install && mkcert -cert-file config/webpack/certificate.pem -key-file config/webpack/key.pem dev.new.expensify.com localhost 127.0.0.1"
   },
   "dependencies": {
-    "@dotlottie/react-player": "^1.6.3",
+    "@dotlottie/react-player": "^1.6.15",
```

Comment:
Note: The changes to this file and the lockfile aren't strictly related to this PR, but it was something that we found and discussed on Slack and it was easy to add the update to my current PR ([ref](https://expensify.slack.com/archives/C01GTK53T8Q/p1701232130505309)).